docs/customization/custom-scripts.md

```diff
@@ -65,6 +65,18 @@ class AnotherCustomScript(Script):
 script_order = (MyCustomScript, AnotherCustomScript)
 ```
 
+The `run()` method is the entrypoint for the script, and it runs in the context of NetBox's own execution environment. This means from here, everything inside NetBox itself is accesible. The [NetBox Shell](../administration/netbox-shell.md) is a good resource to keep in hand, since it allows to see the objects in NetBox in the same way the script does.
```

Comment:
We've already introduced the `run()` method above; this makes no sense to put here. The link to the `nbshell` documentation should be a tip admonition as it doesn't directly relate to authoring a script.